fedimint-api/src/db/mod.rs

```diff
@@ -169,10 +175,86 @@ impl Drop for CommitTracker {
     }
 }
 
+struct IsolatedDatabaseTransaction<'isolated, 'parent: 'isolated> {
+    tx: &'isolated mut DatabaseTransaction<'parent>,
```

Comment:
No need to change it, but IIRC `inner` is preferable from repeating the same name.

Reply:
Changed to inner_tx